test/cypress/integration/04-organization.test.js

```diff
@@ -20,4 +26,37 @@ describe('New organization profile', () => {
     cy.getByDataCy('section-our-team').contains('Our team');
     cy.getByDataCy('section-our-team').contains('Test User Admin');
   });
+
+  describe('Transactions section', () => {
```

Comment:
Moved & adapted this test from `test/cypress/integration/05-user.test.js` to make sure we keep testing the Transactions section